libnetwork/drivers/bridge/errors.go

```diff
@@ -182,6 +182,7 @@ func (ndbee NonDefaultBridgeNeedsIPError) Forbidden() {}
 
 // FixedCIDRv4Error is returned when fixed-cidrv4 configuration
 // failed.
+// TODO(robmry) - unused, remove
```

Comment:
Initially, that's what I did. But, I was asked to put them back, add the TODOs, and deal with them in a later PR - which was fair enough, it was just a drive-by, the types weren't used before this change.

I have that follow-up change ready - along with a couple more, to make `networkConfiguration.Validate()` actually do what it looks like it's doing, and to simplify things on the basis that `config.AddressIPv6` should always have a value.

So, I don't mind ... I can delete them here, or in a separate PR. (Just not both!)

Reply:
Ohh, my bad. I had mistakenly assumed that these types became unused because of changes made in this PR, and was unaware of those prior discussions. Removing them in a separate PR sounds good to me.

Reply:
Ok, removed the TODOs. I've got a change lined up after this one that'll remove the error types.